intranet/apps/users/models.py

```diff
@@ -562,33 +569,43 @@ def grade_number(self) -> int:
 
     @property
     def sex(self) -> str:
-        """Returns "Male" if this user is male, "Female" otherwise.
+        """Returns the gender of this user (male, female, or non-binary).
 
         Returns:
-            "Male" if this user is male, "Female" otherwise.
+            The gender of this user (male, female, or non-binary).
 
         """
-        return "Male" if self.is_male else "Female"
+        return self.gender
```

Comment:
```python
return self.gender or ""
```
In case gender is `None`. I think the serializer is the only thing that uses this field.